android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -330,30 +325,35 @@ private void refreshOpeningTime()
     }
   }
 
-  private void initLocalizedNameView(final View view)
+  private void initNamesView(final View view)
   {
-    mLocalizedNames = (RecyclerView) view.findViewById(R.id.recycler);
-    mLocalizedNames.setNestedScrollingEnabled(false);
-    mLocalizedNames.setLayoutManager(new LinearLayoutManager(getActivity()));
-    mLocalizedNamesAdapter = new MultilanguageAdapter(mParent);
-    mLocalizedNames.setAdapter(mLocalizedNamesAdapter);
-    mLocalizedNamesAdapter.registerAdapterDataObserver(mLocalizedNamesObserver);
-    refreshLocalizedNames();
+    mNamesCaption = (TextView) view.findViewById(R.id.show_additional_names);
+    mNamesCaption.setOnClickListener(this);
+    mAddLanguage = (TextView) view.findViewById(R.id.add_langs);
```

Comment:
Please, group these expressions and separate groups by blank lines. I see following groups:
* mNamesCaption
* mAddLanguage
* mMoreLanguages
* mNames... (i.e. all starting with mNames)